<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>globalgu</class>
 <widget class="QMainWindow" name="globalgu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Projects">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSet_Current_Project_Location"/>
    <addaction name="actionOpen_New_RadTrack_Window"/>
    <addaction name="menuRecent_Projects"/>
    <addaction name="separator"/>
    <addaction name="actionImport_File"/>
    <addaction name="actionExport_Current_Tab"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTabs">
    <property name="title">
     <string>Tabs</string>
    </property>
    <widget class="QMenu" name="menuNew_Tab">
     <property name="title">
      <string>New Tab</string>
     </property>
    </widget>
    <addaction name="menuNew_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Current_Tab"/>
    <addaction name="actionReopen_Closed_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionRename_Current_Tab"/>
   </widget>
   <widget class="QMenu" name="menuExamples">
    <property name="title">
     <string>Examples</string>
    </property>
    <widget class="QMenu" name="menuLoad_Example_Project">
     <property name="title">
      <string>Load Example Project</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuImport_Example_File">
     <property name="title">
      <string>Import Example File</string>
     </property>
    </widget>
    <addaction name="menuLoad_Example_Project"/>
    <addaction name="menuImport_Example_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTabs"/>
   <addaction name="menuExamples"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project ...</string>
   </property>
  </action>
  <action name="actionSet_Current_Project_Location">
   <property name="text">
    <string>Set Current Project Location ...</string>
   </property>
  </action>
  <action name="actionOpen_New_RadTrack_Window">
   <property name="text">
    <string>Open New RadTrack Window ...</string>
   </property>
  </action>
  <action name="actionImport_File">
   <property name="text">
    <string>Import File ...</string>
   </property>
  </action>
  <action name="actionExport_Current_Tab">
   <property name="text">
    <string>Export Current Tab ...</string>
   </property>
  </action>
  <action name="actionClose_Current_Tab">
   <property name="text">
    <string>Close Current Tab</string>
   </property>
  </action>
  <action name="actionReopen_Closed_Tab">
   <property name="text">
    <string>Reopen Closed Tab</string>
   </property>
  </action>
  <action name="actionRename_Current_Tab">
   <property name="text">
    <string>Rename Current Tab ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
